### DISCUSSION TITLE
How to change the style from default to New York afterwards?

### DISCUSSION THREAD

**Discussion by cd-a** [shadcn-ui/ui#3709] [Q&A]
I've already added a few components, but during initialisation I selected the default style. 

Now I want to change to New York.

However, in the docs it says it cannot be changed after initialisation.
![CleanShot 2024-05-10 at 10 12 32@2x](https://github.com/shadcn-ui/ui/assets/75008413/3507db43-0870-429a-9390-a46af9c19faa)

What am I missing here?

**Comment by arey-dev** [comment]
You'd have to re-install your components after changing the style to `new-york`
